ApiFirst/ApiFirst/CustomersValidator.cs

```diff
@@ -0,0 +1,84 @@
+﻿using DomainModels.Models;
+using FluentValidation;
+
+namespace AppServices.Validation
+{
+    public class CustomersValidator : AbstractValidator<Customer>
+    {
+        public CustomersValidator()
+        {
+            RuleFor(x => x.Cpf.Trim().Replace(".", "").Replace("-", "")).MinimumLength(10).MaximumLength(14).Must(BeAValidCpf).
+                NotNull().NotEmpty().WithMessage("Please enter a valid CPF");
+
+            RuleFor(x => x.Email).EmailAddress().Equal(x => x.EmailConfirmation).NotNull().NotEmpty();
+
+            RuleFor(x => x.DateOfBirth).NotEmpty().GreaterThan(DateTime.MinValue).WithMessage("This is required");
+            RuleFor(x => x.FullName).NotEmpty().WithMessage("This is required");
+            RuleFor(x => x.Cellphone).NotEmpty().WithMessage("This is required");
+            RuleFor(x => x.EmailSms).NotEmpty().WithMessage("This is required");
+            RuleFor(x => x.Whatsapp).NotEmpty().WithMessage("This is required");
+            RuleFor(x => x.Country).NotEmpty().WithMessage("This is required");
+            RuleFor(x => x.City).NotEmpty().WithMessage("This is required");
+            RuleFor(x => x.PostalCode).NotEmpty().WithMessage("This is required");
+            RuleFor(x => x.Address).NotEmpty().WithMessage("This is required");
+            RuleFor(x => x.Number).NotEmpty().WithMessage("This is required");
+
+            bool BeAValidCpf(string cpf)
+            {
+
+                int[] multiplicador1 = new int[9] { 10, 9, 8, 7, 6, 5, 4, 3, 2 };
+                int[] multiplicador2 = new int[10] { 11, 10, 9, 8, 7, 6, 5, 4, 3, 2 };
+                string tempCpf;
+                string digito;
+                int soma;
+                int resto;
+
+                cpf = cpf.Trim();
+                cpf = cpf.Replace(".", "").Replace("-", "");
+
+                if (cpf.Length != 11) return false;
+
```

Comment:
A fórmula padrão não exclui cpfs como "111.111.111-11". É possível realizar a exclusão com o seguinte código: 
```suggestion
                if (cpf.Length != 11) return false;
                if(cpf.All(x => x == cpf.First())) return false;

```